test/e2e/run_networking_test.go

```diff
@@ -265,10 +263,10 @@ var _ = Describe("Podman run networking", func() {
 	})
 
 	It("podman run network expose ports in image metadata", func() {
-		session := podmanTest.Podman([]string{"create", "-dt", "-P", nginx})
+		session := podmanTest.Podman([]string{"create", "--name", "test", "-dt", "-P", nginx})
 		session.Wait(90)
 		Expect(session.ExitCode()).To(Equal(0))
-		results := podmanTest.Podman([]string{"inspect", "-l"})
+		results := podmanTest.Podman([]string{"inspect", "test"})
```

Comment:
Should this really change?

Reply:
Yes, we do we care about testing the -l or --latest inspect flag when testing networking code.  We want to test the networking code for remote.